src/components/NavBarComponent.jsx

```diff
@@ -32,7 +32,7 @@ class NavBarComponent extends Component {
   }
 
   render () {
-    const { classes, backToHome, profile } = this.props
+    const { classes, backToHome, profile, actionsPending } = this.props
```

Comment:
Suggestion: using "disabled" param instead of actionsPending. NavBar should not depend on other container's actions. Also, not only createCloudWallet action blocks logout btn, there will also be other actions such as boardcasting tx, unlocking wallet, etc.

Reply:
Also, it's easier for testing